src/queries/text.js

```diff
@@ -9,6 +9,31 @@ import {
   buildQueries,
 } from './all-utils'
 
+function ignoreHiddenElements(container, selector, baseArray) {
+  const hiddenElements = [
+    ...Array.from(container.querySelectorAll("[aria-hidden='true']")),
+  ]
+  baseArray.forEach(base => {
+    if (base.getAttribute('aria-hidden') === 'true') {
+      hiddenElements.push(base)
+    }
+  })
```

Comment:
You can add in the hidden baseArray elements more concisely with an array filter:
```suggestion
  const hiddenElements = [
    ...Array.from(container.querySelectorAll("[aria-hidden='true']")),
    ...baseArray.filter(base => base.getAttribute('aria-hidden') === 'true'),
  ]
```

Reply:
current implementation is just fine.